hypha/apply/funds/tests/test_views.py

```diff
@@ -207,6 +207,30 @@ def test_can_create_project(self):
         self.assertTrue(hasattr(submission, 'project'))
         self.assertEquals(submission.project.id, project.id)
 
+    def test_screen_application_primary_action(self):
```

Comment:
This should be two tests - one with and one without the screening status.

Reply:
ok, updated